src/db.py

```
@@ -1,5 +1,6 @@
 import contextvars
 from typing import Optional
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)